immuni_analytics/celery/authorization_ios/__init__.py

```diff
@@ -10,14 +10,3 @@
 #  GNU Affero General Public License for more details.
 #  You should have received a copy of the GNU Affero General Public License
 #  along with this program. If not, see <https://www.gnu.org/licenses/>.
-
-from immuni_common.models.enums import EnvarEnum
```

Comment:
If there are no other Enums in this file, it may be worth removing it altogether?

Reply:
It's a weird ui problem :D It looks like it merged the deletion of enum.py with the creation of init.py